src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -176,15 +178,17 @@ internal class WamBroker : IBroker
                         _logger,
                         isInteractive: true);
                 }
-                else
-                {
-                    if (IsAadOnlyAuthority(authenticationRequestParameters.Authority))
-                    {
-                        return await AcquireInteractiveWithAadBrowserAsync(
-                            authenticationRequestParameters,
-                            acquireTokenInteractiveParameters.Prompt).ConfigureAwait(false);
-                    }
-                }
+            }
+
+            // no account information available, need an account picker
+
+            if (CanSkipAccountPicker(authenticationRequestParameters.Authority))
+            {
+                _logger.Verbose("[WamBroker] Using AAD plugin account picker");
```

Comment:
Here we are using Browser right? Is it same as AAD plugin account picker?

Reply:
I updated the issue with the some screenshots, to make it clear. 

The "Account Picker" is the Windows control known as AccountsSettingsPane. 
The AAD account picker is an experience hosted in WAM. It can only display Work and School accounts. 

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2289